src/c++/perf_analyzer/report_writer.cc

```diff
@@ -388,4 +407,45 @@ ReportWriter::WriteGpuMetrics(std::ostream& ofs, const Metrics& metric)
   }
 }
 
+void
+ReportWriter::WriteLlmMetrics(std::ostream& ofs)
+{
+  auto [avg_first_token_latency, avg_t2t_latency] = CalculateLlmMetrics();
+  ofs << "," << avg_first_token_latency;
+  ofs << "," << avg_t2t_latency;
+}
+
+std::tuple<double, double>
+ReportWriter::CalculateLlmMetrics()
+{
+  const std::vector<Experiment>& experiments{collector_->GetData()};
+  std::vector<double> first_token_latencies;
+  std::vector<double> t2t_latencies;
+
+  for (const auto& exp : experiments) {
+    for (const auto& req : exp.requests) {
+      for (size_t i = 0; i < req.response_times_.size(); i++) {
+        if (i <= 0) {
+          const std::chrono::duration<double, std::micro> ttft{
```

Comment:
Why `std::micro`? Why not the smallest? (`std::nano`)

Reply:
All the latency related metrics in the csv report were using microseconds, and there's no good way to figure out the unit of each metrics in the file. So I matched to microseconds so that people don't trip. 

For instance, this how the csv file looks like:
```
Concurrency,Inferences/Second,Response Throughput,Client Send,Network+Server Send/Recv,Server Queue,Server Compute Input,Server Compute Infer,Server Compute Output,Client Recv,p50 latency,p90 latency,p95 latency,p99 latency,Avg First Token Latency,Avg Token-to-Token Latency
10,16.6363,499.089,0,520131,867,35,608,6,0,522845,544898,546377,546881,39561.4,16623.7

```